src/DevEntry.js

```diff
@@ -1,18 +1,23 @@
 import React from 'react';
 import { BrowserRouter as Router } from 'react-router-dom';
 import { Provider } from 'react-redux';
-import { init } from './store';
+import { init, RegistryContext } from './store';
 import App from './App';
+import { getBaseName } from '@redhat-cloud-services/frontend-components-utilities/helpers';
 import logger from 'redux-logger';
 
-import getBaseName from './Utilities/getBaseName';
+const TasksDev = () => {
+  const registry = init(logger);
 
-const TasksDev = () => (
-  <Provider store={init(logger).getStore()}>
-    <Router basename={getBaseName(window.location.pathname)}>
-      <App />
-    </Router>
-  </Provider>
-);
+  return (
+    <RegistryContext.Provider value={{ getRegistry: () => registry }}>
```

Comment:
I think we can avoid passing anonymous function as a prop to avoid unwanted [re-rendering](https://user3141592.medium.com/react-gotchas-and-best-practices-2d47fd67dd22#:~:text=Because%20we%20are%20passing%20anonymous,method%20like%20onClick%3D%7Bthis.) by creating a dedicated function:

`const getRegistery = () =>{
       return init(logger); 
 }`

Reply:
I'm having a hard time understanding how to expect to implement this. Do I do this:
```
const TasksDev = () => {
  const getRegistry = () => {
    return init(logger);
  };

  return (
    <RegistryContext.Provider value={{ getRegistry }}>
      <Provider store={registry.getStore()}>
        <Router basename={getBaseName(window.location.pathname)}>
          <App />
        </Router>
      </Provider>
    </RegistryContext.Provider>
  );
};

export default TasksDev;
```
This doesn't work because `registry` is now gone and replacing it with `getRegistry` doesn't work.

Or this?
```
const TasksDev = () => {
  const registry = init(logger);

  const getRegistry = () => {
    return init(logger);
  };

  return (
    <RegistryContext.Provider value={{ getRegistry }}>
      <Provider store={registry.getStore()}>
        <Router basename={getBaseName(window.location.pathname)}>
          <App />
        </Router>
      </Provider>
    </RegistryContext.Provider>
  );
};

export default TasksDev;
```
This also causes the inventory table not to load.

Reply:
This is not important, just thought that it would be an improvement. Anyways, please feel free to ignore this, if it does not work.